modules/update-restricted-network-cli.adoc

```diff
@@ -0,0 +1,186 @@
+                    machineconfiguration.openshift.io/desiredConfig: rendered-worker-a0f873210ec70fb117835a9c9527db29
+                    machineconfiguration.openshift.io/state: Done
+                    machineconfiguration.openshift.io/currentConfig: rendered-master-9194b32791755030dcce887c66024113
+                    machineconfiguration.openshift.io/desiredConfig: rendered-master-9194b32791755030dcce887c66024113
+                    machineconfiguration.openshift.io/state: Done
+
+----
++
+For each machine, the `machineconfiguration.openshift.io/state` must be `Done` before you begin the update.
+
+. Start the cluster update:
++
+----
+$ oc adm upgrade --force --to-image <local_registry_host_name>:<local_registry_host_port>/ocp/release:<release_version> <1>
+----
+<1> For `<local_registry_host_name>` and `<local_registry_host_port>`, specify
+the values that describe your mirror registry host name and port. For
+`<release_version>`, specify the version that you want to upgrade to.
+
+. Review the cluster version status history to monitor the status of the update.
+It might take some time for all the objects to finish updating.
++
+----
+$ oc get clusterversion -o json|jq ".items[0].status.history"
+
+[
+  {
+    "completionTime": null,
+    "image": "quay.io/openshift-release-dev/ocp-release@sha256:9c5f0df8b192a0d7b46cd5f6a4da2289c155fd5302dec7954f8f06c878160b8b",
+    "startedTime": "2019-06-19T20:30:50Z",
+    "state": "Partial",
+    "verified": true,
+    "version": "4.2.1"
+  },
+  {
+    "completionTime": "2019-06-19T20:30:50Z",
+    "image": "quay.io/openshift-release-dev/ocp-release@sha256:b8307ac0f3ec4ac86c3f3b52846425205022da52c16f56ec31cbe428501001d6",
+    "startedTime": "2019-06-19T17:38:10Z",
+    "state": "Completed",
+    "verified": false,
+    "version": "4.2.0"
+  }
+]
+----
++
+The history contains a list of the most recent versions applied to the cluster.
+This value is updated when the CVO applies an update. The list is ordered by
+date, where the newest update is first in the list. Updates in the history have
+state `Completed` if the rollout completed and `Partial` if the update failed
+or did not complete.
++
+[IMPORTANT]
+====
+If an upgrade fails, the cluster version reports a condition that explains the
+failure. Contact Red Hat support.
+====
+
+. After the update completes, you can confirm that the cluster version has
+updated to the new version:
++
+----
+$ oc get clusterversion
+
+NAME      VERSION     AVAILABLE   PROGRESSING   SINCE     STATUS
+version   4.2.1       True        False         2m        Cluster version is 4.2.1
+----
+
+. Delete the original `ImageContentSourcePolicy` resource, if one existed:
++
+----
+$ oc delete ImageContentSourcePolicy <name> <1>
+----
+<1> For `<name>`, specify the name of the original `ImageContentSourcePolicy` resource.
+
+. If you modified the contents of the `repositoryDigestMirrors` section of the
```

Comment:
The part is already covered by "oc delete ImageContentSourcePolicy" command in the above section, should be removed.